src/main/java/Storage.java

```diff
@@ -0,0 +1,101 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Storage {
```

Comment:
public class and methods undocumented bro, small issue, maybe u will do it later haha.